flipperdevices/flipperzero-firmware
# [FL-2481] Renamed assets->resources; enforcing Manifest build if it does not exist

# What's new

- [FL-2481] Updated make rules to automatically rebuild resources Manifest if it does not exist
- Now storing resources/Manifest in git repo
- Only updating it if its files actually changed

# Verification 

- `make update_bundle`
- change resources or remove manifest
- `make update_bundle` again, verify that Manifest is rebuilt and placed in resources.tar

- Ensure that multiple `make manifest` runs don't actually change the Manifest file, if there were no changes to original files

# Checklist (For Reviewer)



[FL-2481]: https://flipperzero.atlassian.net/browse/FL-2481?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
